lib/src/widgets/information_message/ds_information_message.widget.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+
+import 'package:blip_ds/blip_ds.dart';
+
+/// Messages and warnings to be shown in agent and customer actions
+///
+/// Parameters [chatbotIdentity] and [agentIdentity] are sent to personalize the message.
+/// Depending on the message type defined by the [messageType] parameter, other parameters
+/// may or may not be mandatory.
+class DSInformationMessage extends StatelessWidget {
```

Comment:
I recommend that you rename this widget, suggest: `DSTicketMessage`, because the messages are related with the ticket.